packages/common/src/workflow-options.ts

```diff
@@ -38,24 +41,76 @@ export enum WorkflowIdReusePolicy {
   WORKFLOW_ID_REUSE_POLICY_REJECT_DUPLICATE = 3,
 
   /**
-   * Terminate the current workflow if one is already running.
+   * Terminate the current workflow if one is already running, otherwise allow a duplicate ID.
+   *
+   * Deprecated: use `WORKFLOW_ID_CONFLICT_POLICY_TERMINATE_EXISTING` and `WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE` instead.
+   *
+   * This option belongs in `WorkflowIdConflictPolicy` but it is here for backwards compatibility.
+   * *Note: If setting this option, `WorkflowIdConflictPolicy` must be left unspecified.*
    */
   WORKFLOW_ID_REUSE_POLICY_TERMINATE_IF_RUNNING = 4,
 }
 
 checkExtends<temporal.api.enums.v1.WorkflowIdReusePolicy, WorkflowIdReusePolicy>();
 checkExtends<WorkflowIdReusePolicy, temporal.api.enums.v1.WorkflowIdReusePolicy>();
 
+// Avoid importing the proto implementation to reduce workflow bundle size
+// Copied from temporal.api.enums.v1.WorkflowIdConflictPolicy
+/**
+ * Defines what happens when trying to start a Workflow with the same ID as a *Running* Workflow.
+ *
+ * See {@link WorkflowOptions.workflowIdReusePolicy} for what happens when trying to start a Workflow with the same ID as a *Closed* Workflow.
+ *
+ * *Note: It is not possible to have two actively running Workflows with the same ID.*
+ */
+export enum WorkflowIdConflictPolicy {
```

Comment:
I really want to remove the `WORKFLOW_ID_CONFLICT_POLICY_` prefix on these values.

I know this is inconsistent with the Reuse Policy enum just above, but `WorkflowIdReusePolicy` is the broken one and we'll need to fix it in a backward-compatible way. I'm aware of three other enums that follow that pattern.

But in all other TS APIs introduced in the last two years, we specifically avoided those prefixes; and more recently, [we even agreed](https://github.com/temporalio/sdk-typescript/pull/1156/files#r1256407894) that we'd try to transition to string unions as much as possible, though we didn't make progress on achieving that.

Reply:
@mjameswh @antlai-temporal is this comment the only thing blocking this PR? Is there a path forwards where we get this PR merged now, without blocking on https://github.com/temporalio/sdk-typescript/pull/1534?